Automattic/jetpack
# Untangling Cleanup: Clean up `wpcom_is_nav_redesign_enabled()` flag

Related to: https://github.com/Automattic/dotcom-forge/issues/7927

## Proposed changes:

* As we've launched untangling & nav redesign, the `wpcom_is_nav_redesign_enabled` function name is not relevant anymore and can be confusing for future developers. 

This goes alongside this wpcom PR: D154312-code

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch on your sandbox
* On a site using the classic admin interface style, set the `wpcom_classic_tour_completed` option to `'0'`:
  * On your sandbox, run `wpsh` and then `update_blog_option( SITE_ID, 'wpcom_classic_tour_completed', '0' );`
* Go to `/wp-admin/?admin-interface-changed=true`
* Check that you see the tour dialog:

![image](https://github.com/Automattic/jetpack/assets/8511199/58f4fead-b00b-4961-bbf7-b642b80e4253)
